diff --git a/src/targets.py b/src/targets.py
index 00e2bde..2b077e8 100644
--- a/src/targets.py
+++ b/src/targets.py
@@ -69,6 +69,7 @@ registry-{stage}-{name}:
 \t  --tag $(REGISTRY_USERNAME)/{stage}-{name}:{version} \\
 \t  --tag $(REGISTRY_USERNAME)/{stage}-{name}:$(TAG) \\
 \t  --tag $(REGISTRY_USERNAME)/{stage}-{name}:latest \\
+\t  --tag ttl.sh/siderolabs/stagex/{stage}-{name}:latest \\
 \t  --provenance=false \\
 \t  --build-arg SOURCE_DATE_EPOCH=1 \\
 \t  --build-arg BUILDKIT_MULTI_PLATFORM=1 \\
